### PR TITLE
Added sound_of_silence/3

### PR DESCRIPTION
**Pull request by nobikik9**

### CODE REVIEW

File: lib/membrane_caps_audio_raw.ex [hunk 1]
```diff
@@ -283,6 +283,17 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+  """
+  @spec sound_of_silence(Time.non_neg_t(), t, (float -> integer)) :: binary
+  def sound_of_silence(time, %__MODULE__{} = caps, round_f \\ &(&1 |> :math.ceil() |> trunc))
```
Comment (bblaszkow06) [inline lib/membrane_caps_audio_raw.ex]: I would change the arguments order to be consistent with `sound_od_silence/1` - caps should go first followed by the time and rounding function.

File: lib/membrane_caps_audio_raw.ex [hunk 2]
```diff
@@ -283,6 +283,17 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
```
Comment (bblaszkow06) [inline lib/membrane_caps_audio_raw.ex]: Maybe adding a unit test for this function would be an overkill, but some example (working as doctest) is welcome

Comment (bblaszkow06) [reply lib/membrane_caps_audio_raw.ex]: For reference see https://hexdocs.pm/ex_unit/ExUnit.DocTest.html and https://github.com/membraneframework/bunch/blob/v0.1.1/lib/bunch/enum.ex#L262

File: lib/membrane_caps_audio_raw.ex [hunk 3]
```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
...
+
+  ## Examples:
+  The following code generates one second of the silence for the given caps
```
Comment (mat-hek) [inline lib/membrane_caps_audio_raw.ex]: It actually does generate 100 microseconds ;) I bet you changed this below and forgot to change here, and that's why such comments should be avoided

File: lib/membrane_caps_audio_raw.ex [hunk 4]
```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
...
+
+    iex> alias Membrane.Caps.Audio.Raw, as: Caps
+    Membrane.Caps.Audio.Raw
```
Comment (mat-hek) [inline lib/membrane_caps_audio_raw.ex]: This line can be removed, results are necessary only when they need to be tested

File: lib/membrane_caps_audio_raw.ex [hunk 5]
```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
...
+    iex> caps = %Caps{sample_rate: 48_000, format: :s16le, channels: 2}
+    %Membrane.Caps.Audio.Raw{channels: 2, format: :s16le, sample_rate: 48000}
```
Comment (mat-hek) [inline lib/membrane_caps_audio_raw.ex]: as above

File: lib/membrane_caps_audio_raw.ex [hunk 6]
```diff
@@ -283,6 +283,27 @@ defmodule Membrane.Caps.Audio.Raw do
...
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+
+  ## Examples:
+  The following code generates one second of the silence for the given caps
+
+    iex> alias Membrane.Caps.Audio.Raw, as: Caps
```
Comment (mat-hek) [inline lib/membrane_caps_audio_raw.ex]: Examples should be enclosed in \``` quotes, so they were properly formatted in docs. Generate docs with `mix docs` to check

File: lib/membrane_caps_audio_raw.ex [hunk 7]
```diff
@@ -283,6 +283,25 @@ defmodule Membrane.Caps.Audio.Raw do
...
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+
+  ## Examples:
+  The following code generates one second of the silence for the given caps
+
+      iex> alias Membrane.Caps.Audio.Raw, as: Caps
```
Comment (mat-hek) [inline lib/membrane_caps_audio_raw.ex]: Examples should be enclosed in \``` quotes, so they were properly formatted in docs. Generate docs with `mix docs` to check

**Pull request merged**